playground/src/components/NavBar.tsx

```diff
@@ -6,7 +6,10 @@ const NavBar = () => {
   return (
     <>
       <div className="flex items-center justify-between flex-wrap bg-blue-700 p-6">
-        <div className="flex items-center flex-shrink-0 text-white mr-6" data-testid="logo">
+        <div
```

Comment:
This is only made prettier using `npx prettier --write .`. I wasn't able to undo that step. I can manually remove those files from the PR if you want.